models/engine/db_storage.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/python3
 """
-Contains the class DBStorage
+Contains the class DBStorage to save in database
```

Comment:
What type of objects are returned in the objects list? and please check the comments in the count function

Reply:
we are returning all objects for each class (User, Amenity, City, Place, State, Review) and we are going to do better our comments for next one